Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM php:8.1
```

Comment:
I didn't find a "official" image with php and composer. The image provided by the [composer team](https://hub.docker.com/_/composer?tab=description&page=1) is not a good choice for us, because we must have a clear view/control about which PHP version we're running (check the Troubleshooting section in the link below for more info).

[composer troubleshooting docker hub](https://hub.docker.com/_/composer?tab=description&page=1)

Reply:
Wow, that's a really good idea! Thanks for teaching me this trick! :D